dockerfiles/interop/src/env.yaml

```diff
@@ -0,0 +1,26 @@
+application:
+  hostname: REPLACE_HOSTNAME
+  ocphostname: REPLACE_OCP_HOSTNAME
+  ocpsecurehostname: REPLACE_OCP_SECURE_HOSTNAME
+  password: password
+  user: mtr
+  version: 1.0
```

Comment:
We may want to make this a variable, or maybe we'd want a different env.yaml file for each future version?

Reply:
@ascerra Good catch, I will make that change